catalog/ui/search-ui/standard/src/main/webapp/js/controllers/openlayers.controller.js

```diff
@@ -115,18 +115,17 @@ define(['underscore',
             setupEvents: function () {
                 var controller = this;
                 this.mapViewer.on('click', function(event) {
-                    var feature;
-
-                    controller.mapViewer.forEachLayerAtPixel(event.pixel, function (layer) {
-                        feature = controller.mapViewer.forEachFeatureAtPixel(event.pixel,
-                            function(feature) {
-                                controller.trigger('click:left', feature);
-                            }, this, function(testLayer) {
-                                if (testLayer === layer) {
-                                    return true;
-                                }
-                            });
-                    });
+                    controller.mapViewer.forEachFeatureAtPixel(event.pixel,
```

Comment:
Won't this still select multiple metacards if they are in the same pixel/location?  If so, you might have to keep state in the closure about if a metacard has been found yet and only trigger if none have been found yet.